<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptRecorderWidget</class>
 <widget class="QDockWidget" name="ScriptRecorderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Script Recorder</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <widget class="QPushButton" name="changeFolderButton">
      <property name="text">
       <string notr="true">Change folder</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="folderLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string notr="true">Folder:</string>
      </property>
      <property name="buddy">
       <cstring>changeFolderButton</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string notr="true">Script name:</string>
      </property>
      <property name="buddy">
       <cstring>scriptNameEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="recordButton">
      <property name="text">
       <string notr="true">Record</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="replayButton">
      <property name="text">
       <string notr="true">Replay</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="scriptNameEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>changeFolderButton</tabstop>
  <tabstop>scriptNameEdit</tabstop>
  <tabstop>recordButton</tabstop>
  <tabstop>replayButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
